src/card/__tests__/card.test.js

```diff
@@ -12,7 +12,7 @@ import {header as headerImg, thumbnail as thumbnailImg} from '../images';
 
 test('Card - basic functionality', () => {
   const props = {
-    action: <a href="#">Link to a Place&nbsp;&nbsp;&nbsp;&gt;</a>,
+    action: <a href="#test">Link to a Place&nbsp;&nbsp;&nbsp;&gt;</a>,
```

Comment:
The `<a href="#">` links should still be allowed. Maybe we can override some rules for the plugin?